src/helper-functions.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * finds array index of array object which matches predicate
+ * @param array {Array}
+ * @param predicate {Function}
+ * @returns {number} || {undefined}
+*/
+export const findIndex = (array: any[], predicate: Function) => {
+  const length = array == null ? 0 : array.length;
+  if (!length) {
+    return -1;
+  }
+  let index = -1;
+  while (++index < length) {
+    if (predicate(array[index])) {
+      return index;
+    }
+  }
+  return -1;
+};
+
+/**
+ * Removes false and null values from array
+ * @param array {Array}
+ * @returns {Array}
+ */
+export const compact = (array: any[]) => {
+  let index = 0;
+  const result: any[] = [];
+
+  for (const value of array) {
+    if (value) {
+      result[index++] = value;
+    }
+  }
+  return result;
+};
+
+/**
+ * finds an array elements which matches the predicate
+ * @param array {Array}
+ * @param predicate {Function}
+ * @returns {any} || {undefined}
+*/
+export const find = (array: any[], predicate: Function) => {
+  const length = array == null ? 0 : array.length;
+  if(!length) {
+    return undefined;
+  }
+  let index = -1;
+  while (++index < length) {
+    if (predicate(array[index])) {
+      return array[index];
+    }
+  }
+  return undefined;
+};
+
+/**
+ * compares OpenrpcDocuments
+ *
+ * @param doc1 {OpenrpcDocument}
+ * @param doc2 {OpenrpcDocument}
+ * @returns {boolean}
+*/
+export const rpcDocIsEqual = (doc1: any, doc2: any) => {
```

Comment:
If it's only included in the test, you can put it in a `.test.` file to have it not included in the build.

Reply:
https://github.com/open-rpc/schema-utils-js/blob/master/package.json#L17